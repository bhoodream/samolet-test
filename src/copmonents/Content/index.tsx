import { FC } from 'react';
import { Row, Col } from 'antd';

const Content: FC = (props) => {
    return (
        <Row>
            <Col xs={{ span: 22, offset: 1 }} sm={{ span: 20, offset: 2 }}>
                {props.children}
            </Col>
        </Row>
    );
};

export default Content;
