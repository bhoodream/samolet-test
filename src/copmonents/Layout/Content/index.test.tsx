import renderer from 'react-test-renderer';
import Content from './index';

it('renders correctly', () => {
    const tree = renderer.create(<Content>Contentt</Content>).toJSON();

    expect(tree).toMatchSnapshot();
});
