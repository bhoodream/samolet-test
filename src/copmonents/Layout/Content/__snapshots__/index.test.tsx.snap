// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-row"
  style={Object {}}
>
  <div
    className="ant-col ant-col-xs-22 ant-col-xs-offset-1 ant-col-sm-20 ant-col-sm-offset-2"
    style={Object {}}
  >
    Contentt
  </div>
</div>
`;
